### PR TITLE
Add missing 1.4 news note for new function pkgdir() (#33128)

### PR DESCRIPTION
**Pull request by IanButterworth**
Apologies, I missed this out when doing https://github.com/JuliaLang/julia/pull/33128

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ New library functions
 * `takewhile` and `dropwhile` have been added to the Iterators submodule ([#33437]).
 * `accumulate` has been added to the Iterators submodule ([#34033]).
 * There is a now an `evalpoly` function meant to take the role of the `@evalpoly` macro. The function is just as efficient as the macro while giving added flexibility, so it should be preferred over `@evalpoly`. `evalpoly` takes a list of coefficients as a tuple, so where one might write `@evalpoly(x, p1, p2, p3)` one would instead write `evalpoly(x, (p1, p2, p3))` ([#32753]).
+* `pkgdir(ModuleName)` now provides an simpler way to return the package root directory of a module (or submodule) than the typically used `dirname(dirname(pathof(ModuleName)))` ([#33128]).
```
Comment (dkarrasch) [inline NEWS.md]: "a simpler way"

**Review by dkarrasch — COMMENTED** [review]
Just one typo. I believe you will need to add the hyperlink to the original PR at the very bottom of the file manually, since this is against the release branch, and not master. But I'm not sure...

**Comment by ViralBShah** [comment]
Should this go on master and get backported to 1.4?

**Comment by ianshmean** [comment]
I believe so. But i think @dkarrash is right, I need to add the url to the bottom of the file to make the link right. I can do that in an hour or so.

**Comment by dkarrasch** [comment]
> Should this go on master and get backported to 1.4?

I'm not sure  @ViralBShah. On master, we have a NEWS file for v1.5. How would we merge this and then backport to v1.4?

**Comment by ianshmean** [comment]
This still needs to be resolved. Missing from the rc2 NEWS

File: NEWS.md [hunk 2]
```diff
@@ -33,6 +33,7 @@ New library functions
 * `takewhile` and `dropwhile` have been added to the Iterators submodule ([#33437]).
 * `accumulate` has been added to the Iterators submodule ([#34033]).
 * There is a now an `evalpoly` function meant to take the role of the `@evalpoly` macro. The function is just as efficient as the macro while giving added flexibility, so it should be preferred over `@evalpoly`. `evalpoly` takes a list of coefficients as a tuple, so where one might write `@evalpoly(x, p1, p2, p3)` one would instead write `evalpoly(x, (p1, p2, p3))` ([#32753]).
+* `pkgdir(ModuleName)` now provides a simpler way to return the package root directory of a module (or submodule) than the typically used `dirname(dirname(pathof(ModuleName)))` ([#33128]).
```
Comment (fredrikekre) [inline NEWS.md]:
```suggestion
* `pkgdir(ModuleName)` now provides a simpler way to return the package root directory of a module (or submodule) than the typically used `dirname(dirname(pathof(ModuleName)))` ([#33128]).
```

**Pull request merged**